### PR TITLE
Update terms doc. to most recent version from google docs.

### LINKED ISSUE #154: Update Terms & conditions
Update Terms and conditions

A new paragraph has been added to Terms and conditions in Docs: Data retention and deletion. This paragraph must be included in the Terms and conditions that can be viewed in the Desktop Wallet.

### PR DESCRIPTION
**Pull request by soerenbf**
## Purpose

Update terms and conditions to reflect most recent version in google docs. Changes are a reflection of mostly differences in what is exported by google docs. No corrections have been made to typography, as that should come from the source (google docs) rather than changes to the export - to keep the update flow as simple as possible.

Closes #154 

## Changes
- updated terms document with fresh export from google docs.

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [x] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

**Comment by orhoj** [comment]
There are two typos in the google document. I have added suggestions in that document to fix them, and I think we should update that document (do we need to wait for Lotte to fix things in there?), so that we can generate the terms and conditions without the new typos.

**Comment by soerenbf** [comment]
> There are two typos in the google document. I have added suggestions in that document to fix them, and I think we should update that document (do we need to wait for Lotte to fix things in there?), so that we can generate the terms and conditions without the new typos.

I think that's the better way to do it as well to ensure a single source of truth. I don't know the procedure of updating that document however.. Maybe we should wait for Lotte to reply?

**Comment by soerenbf** [comment]
Terms have been revised in google docs by Lotte, and updated with the new version here.

File: resources/html/terms.html [hunk 1]
```diff
@@ -419,19 +402,19 @@
...
+    <body class="c5">
+        <p class="c26 title" id="h.1h19t7261en8">
             <span class="c23"
                 >Terms and conditions for Concordium Desktop Wallet</span
             >
         </p>
-        <h1 class="c20">
-            <span class="c6">Effective from June 09, 2021</span>
+        <h1 class="c11">
+            <span class="c3">Effective from June 09, 2021</span>
```
Comment (orhoj) [inline resources/html/terms.html:412]: This date has to be updated as the document has changed.

Comment (soerenbf) [reply resources/html/terms.html:412]: Let's do this one parallel to the docs change...

Comment (soerenbf) [reply resources/html/terms.html]: I've notified Lotte, and she's OK with doing the change in parallel.

**Review by orhoj — APPROVED** [review]

**Pull request merged**